.github/workflows/test.yml

```
@@ -31,6 +31,28 @@ jobs:
           path: ~/go/bin
           key: ${{ runner.os }}-go-tparse-binary
 
+  build-architectures:
```

I think just `build` is fine. @marbar3778 wdyt?

```suggestion
  build:
```